site-manage/site-manage-tool/tool/src/webapp/js/site-manage.js

```diff
@@ -8,69 +8,75 @@ $.ajaxSetup({
 });
 
 /*
- calling template has dom placeholder for dialog,
- args:class of trigger, id of dialog, message strings
+ calling template has dom placeholder for one or more dialogs,
+ args:class of trigger(s), id of dialog (will get a site id suffix), message strings
  */
 sakai.getSiteInfo = function(trigger, dialogTarget, nosd, nold){
-  $("." + trigger).click(function(e){
-    e.preventDefault();
-    bootstrap.Modal.getOrCreateInstance(document.getElementById(dialogTarget)).show();
-  });
-  const siteId = $("." + trigger).attr('id');
-  if (!siteId) {
-    return;
-  }
-  const siteURL = `/direct/site/${siteId}/info.json`;
-  jQuery.getJSON(siteURL, function (data) {
-    var desc = '', shortdesc = '', title = '', owner = '', email = '';
-    if (data.description) {
-      desc = unescape(data.description);
-    }
-    else {
-      desc = nold;
-    }
-    if (data.shortDescription) {
-      shortdesc = data.shortDescription;
-    }
-    else {
-      shortdesc = nosd;
+  $("." + trigger).each(function(){
+    const siteId = $(this).attr('id');
+    if (!siteId) {
+      return;
     }
 
-    if (data.contactName) {
-      owner = data.contactName;
-    }
+    const dialogTargetSite = dialogTarget + "_" + siteId;
 
-    if (data.contactEmail) {
-      email = " (<a href=\"mailto:" + data.contactEmail.escapeHTML() + "\" id=\"email\">" + data.contactEmail.escapeHTML() + "</a>)";
-    }
+    $(this).click(function(e){
+       e.preventDefault();
+       bootstrap.Modal.getOrCreateInstance(document.getElementById(dialogTargetSite)).show();
+     });
 
-    if (data.props) {
-      if (data.props['contact-name']) {
-        owner = data.props['contact-name'];
+    const siteURL = `/direct/site/${siteId}/info.json`;
+    jQuery.getJSON(siteURL, function (data) {
+      var desc = '', shortdesc = '', title = '', owner = '', email = '';
```

Comment:
should be let instead of var?

Reply:
FYI, that part of the code didn't technically change.  It was existing code that just got indented because of the addition of the function at line 15.  Happy to change it if it's desired though.